examples/http_body/README.md

```diff
@@ -0,0 +1,9 @@
+## http_headers
```

Comment:
```suggestion
## http_body
```

Reply:
I've addressed this suggestion in my next commit.